README.md

```diff
@@ -59,6 +59,8 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
 ```
 
+**Note:** You can add `--no-use` to the end of the above script (...`nvm.sh --no-use`) to postpone using `nvm` until you manually `use` it (e.g. `nvm use default`).
```

Comment:
or just `nvm use` by itself :-)

This is great tho, I'll tweak and squash it prior to merging.

Reply:
`nvm use` also uses the default if available; `nvm use default` won't actually work for all users either.

`nvm use node` will work if there's a version installed - if not, `nvm install node` will both install and use node. I don't think there's a one-size-fits-all command, so maybe something generic that links to the [usage](https://github.com/creationix/nvm#usage) section?

I'd probably go with the same commit message but prefixed with `[Docs] ` :-)